documentation.md

```diff
@@ -104,7 +104,7 @@ const crawler = new fdir().withDirs();
 
 ### `withSymlinks({ resolvePaths: boolean })`
 
-Use this to follow all symlinks recursively.
+Use this to follow all symlinks recursively. Not available with relative paths on unless the parameter is set to `false`.
```

Comment:
I will have to take a deep look into this and see if there's a way around.

Reply:
i don't think it's possible to have it available with realpaths as realpaths can point to anywhere outside the root, unless you add a loop to add the necessary `../`s and rework the relative path system, which sounds really complex to do